File: packages/javascript/lib/string/strCount.ts
export function strCount(str: string, search: string): number {
	if (!str || !search) {
		return 0;
	}

	let count = 0;
	let pos = str.indexOf(search);

	while (pos > -1) {
		count += 1;
		pos = str.indexOf(search, (pos += search.length));
	}

	return count;
}

File: packages/python/qsu/string/strCount.py
def strCount(str: str, search: str) -> int:
	if not str or not search:
		return 0

	count = 0
	pos = str.find(search)

	while pos > -1:
		count += 1
		pos = str.find(search, pos + len(search))

	return count
